### ISSUE TITLE
Pester 5 cannot run tests

### ISSUE THREAD

**Issue by RichieBzzzt** [DataThirstLtd/azure.databricks.cicd.tools#134]
I know there's issues around Pester 5 running older test suites, but the _InvokeTests specifies a minimum version of 4.4.2. Could we set a maximumversion of 4.9.0 as well until someone somewhere has the spare time to re-write the tests.

**Comment by simondmorias** [comment]
Agreed - the build isn't running tests right now because of this. I will pickup towards the end of next week.

**Comment by RichieBzzzt** [comment]
LMK if you want a hand.
